<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editor</class>
 <widget class="QWidget" name="editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>691</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="view_prog_b">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>280</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Программы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_dir_b">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>310</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Директории</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_file_b">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>340</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Файлы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="view_user_b">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Пользователи</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Показать</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_name_le">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="p_date_de">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>110</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_platform_le">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>270</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="p_apppo_le">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>270</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="p_add_b">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/add.png</normaloff>C:/Users/Vladimir/Desktop/add.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата выхода</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Платформа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>250</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Назначение</string>
   </property>
  </widget>
  <widget class="QComboBox" name="d_prog_cb">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="d_name_le">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="d_add_b">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>320</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/add.png</normaloff>C:/Users/Vladimir/Desktop/add.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_name_le">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_ext_le">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>370</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_size_le">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="f_date_de">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>81</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="f_dir_cb">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="f_add_b">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/add.png</normaloff>C:/Users/Vladimir/Desktop/add.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="u_name_le">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>221</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="u_country_le">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="u_prog_cb">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="u_add_b">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>420</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/add.png</normaloff>C:/Users/Vladimir/Desktop/add.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Программа</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Директория</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Директория</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Расширение</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>350</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Создан</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Размер</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Полное имя</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>400</y>
     <width>49</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Страна</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>400</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Программа</string>
   </property>
  </widget>
  <widget class="QPushButton" name="p_change_b">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>270</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/pencil.png</normaloff>C:/Users/Vladimir/Desktop/pencil.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="d_change_b">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>320</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/pencil.png</normaloff>C:/Users/Vladimir/Desktop/pencil.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="f_change_b">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>370</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/pencil.png</normaloff>C:/Users/Vladimir/Desktop/pencil.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="u_change_b">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>420</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>C:/Users/Vladimir/Desktop/pencil.png</normaloff>C:/Users/Vladimir/Desktop/pencil.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
